frontend/test/metabase/scenarios/admin/settings/public-sharing.cy.spec.js

```diff
@@ -170,7 +225,14 @@ describe("scenarios > admin > settings > public sharing", () => {
       cy.visit("/admin/settings/public-sharing");
     });
 
-    cy.findByText(expectedActionName).should("not.have.attr", "href");
+    cy.then(function () {
```

Comment:
I think we use arrow functions for `then` callbacks everywhere, should we follow that pattern here as well?

Reply:
This is the only place that won't work with that. https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Avoiding-the-use-of-this

I could `cy.get("@alias")` just fine, but in this case, there were 2 aliases that I need to access, so if I use `cy.get` I need to nest them which would look worse. So, I opt for `cy.then` + `this.alias` in this case.

Reply:
Thanks for calling this out btw. I think I should have added a comment on this. Since it's not that apparent why an anonymous `function` is being used.